protocol_rfcs/geospatial-types.md

```diff
@@ -0,0 +1,89 @@
+* `vincenty`: [https://en.wikipedia.org/wiki/Vincenty%27s_formulae](https://en.wikipedia.org/wiki/Vincenty%27s_formulae)
+* `thomas`: Thomas, Paul D. Spheroidal geodesics, reference systems, & local geometry. US Naval Oceanographic Office, 1970.
+* `andoyer`: Thomas, Paul D. Mathematical models for navigation systems. US Naval Oceanographic Office, 1965.
+* `karney`: [Karney, Charles FF. "Algorithms for geodesics." Journal of Geodesy 87 (2013): 43-55](https://link.springer.com/content/pdf/10.1007/s00190-012-0578-z.pdf), and [GeographicLib](https://geographiclib.sourceforge.io/)
+
+### Per-file statistics
+
+For geospatial types, the most useful statistics for data skipping are **bounding boxes**. A bounding box is a minimal axis-aligned rectangle, parallelepiped, or hyper-parallelepiped, that fully contains all points of all geometries or geographies in a given set. It is typically represented by two points: the **lower-left** and **upper-right** corners. Each point must include **X** and **Y** coordinates, but may also optionally include **Z** (elevation) and **M** (measure) values.
+
+Since a bounding box naturally defines spatial **minimum** and **maximum** values, we can leverage the existing min and max column statistics fields to store the points. The values are encoded in **[Well-Known Text (WKT)](https://libgeos.org/specifications/wkt/)** format. For example:
+
+```json
+{
+  "numRecords": 23,
+  "minValues": {
+    "geometryCol": "POINT(-122.419 37.774)"
+  },
+  "maxValues": {
+    "geometryCol": "POINT(-120.503 38.021)"
+  },
+  "nullCount": {
+    "geometryCol": 0
+  }
+}
+```
+
+
+> ***Add new rows in the [Primitive Types](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#primitive-types) table.***
+
+### Primitive Types
+| Type Name | Description                                                                                                                                                              |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| geometry  | [WKB](https://libgeos.org/specifications/wkb/#standard-wkb) encoded byte array with a given CRS (Coordinate Reference System) value.                                     |
+| geography | [WKB](https://libgeos.org/specifications/wkb/#standard-wkb) encoded byte array with a given CRS (Coordinate Reference System) value and an edge interpolation algorithm. |
+
+### Geospatial data in Parquet
+
+The Parquet format now includes [native support for geospatial data](https://github.com/apache/parquet-format/commit/94b9d631aef332c78b8f1482fb032743a9c3c407). This update introduces standardized **geospatial logical types**, which are encoded using the BYTE_ARRAY physical type. In addition, a new `GeospatialStatistics` field has been added to support storing **bounding boxes** in the row group statistics.
```

Comment:
Add this information to the https://github.com/delta-io/delta/blob/master/PROTOCOL.md#delta-data-type-to-parquet-type-mappings